### ISSUE TITLE
Unable to run power-management/poweroff or cold/warm-boot-stress-test under checkbox-cli 2.9.0

### ISSUE THREAD

**Issue by vanessa525** [canonical/checkbox#658]
### Bug Description

Unable to run power-management/poweroff or cold/warm-boot-stress-test under checkbox-cli 2.9.0

The test jobs power-management/poweroff of client-cert-desktop-22-04-automated and cold-boot-stress-test, warm-boot-stress-test of client-cert-desktop-22-04-stress cannot be execution

The com.canonical.certification::power-management/poweroff cannot do the test
```
-------------------------[ power-management/poweroff ]--------------------------
ID: com.canonical.certification::power-management/poweroff
Category: com.canonical.plainbox::power-management
Waiting for the system to shut down or reboot...
Traceback (most recent call last):
  File "/usr/bin/checkbox-cli", line 8, in <module>
    sys.exit(main())
  File "/usr/lib/python3/dist-packages/checkbox_ng/launcher/checkbox_cli.py", line 163, in main
    subcmd.invoked(ctx)
  File "/usr/lib/python3/dist-packages/checkbox_ng/launcher/subcommands.py", line 261, in invoked
    self._run_jobs(self.ctx.sa.get_dynamic_todo_list())
  File "/usr/lib/python3/dist-packages/checkbox_ng/launcher/stages.py", line 375, in _run_jobs
    builder = self._run_single_job_with_ui_loop(
  File "/usr/lib/python3/dist-packages/checkbox_ng/launcher/stages.py", line 189, in _run_single_job_with_ui_loop
    result_builder = self.sa.run_job(job.id, ui, False)
  File "/usr/lib/python3/dist-packages/plainbox/impl/decorators.py", line 142, in wrapper
    raise exc
  File "/usr/lib/python3/dist-packages/plainbox/impl/decorators.py", line 136, in wrapper
    return func(*args, **kwargs)
  File "/usr/lib/python3/dist-packages/plainbox/impl/session/assistant.py", line 1486, in run_job
    self._restart_cmd_callback(
  File "/usr/lib/python3/dist-packages/checkbox_ng/launcher/subcommands.py", line 312, in <lambda>
    lambda session_id: [shlex.join(respawn_cmd.append(session_id))]
  File "/usr/lib/python3.10/shlex.py", line 320, in join
    return ' '.join(quote(arg) for arg in split_command)
TypeError: 'NoneType' object is not iterable
```

### To Reproduce

1. Install checkbox-cli 2.9
2. Run checkbox-cli run com.canonical.certification::cold-boot-stress-test or com.canonical.certification::warm-boot-stress-test or com.canonical.certification::power-management/poweroff

Expected result: The test can be finished

Actual output: it cannot executant continue 
 

### Environment

OS: Desktop image 22.04
Checkbox type: Debian
Checkbox version: 2.9.0-ubuntu22.04.1

**Comment by kissiel** [comment]
Thank you for the bug report, this got just fixed. The PR is here: https://github.com/canonical/checkbox/pull/657
This will land very soon.

**Comment by pieqq** [comment]
The patch has landed and Checkbox v2.9.1 is available in the beta channels, so I'm marking this as Closed.
